src/middlewared/middlewared/alert/source/freenas_bmc.py

```diff
@@ -3,15 +3,23 @@
 
 from freenasUI.common.pipesubr import pipeopen
 
-from middlewared.alert.base import Alert, AlertLevel, ThreadedAlertSource
+from middlewared.alert.base import AlertClass, AlertCategory, AlertLevel, Alert, ThreadedAlertSource
 
 logger = logging.getLogger("FreeNASBMCAlert")
 
 
-class FreeNASBMCAlertSource(ThreadedAlertSource):
+class FreeNASBMCAlertClass(AlertClass):
+    category = AlertCategory.HARDWARE
     level = AlertLevel.CRITICAL
     title = "FreeNAS Mini Critical IPMI Firmware Update Available"
+    text = (
+        "Your FreeNAS Mini has an available IPMI firmware update, please click "
+        "<a href=\"https://support.ixsystems.com/index.php?/Knowledgebase/Article/View/287\" target=\"_blank\">"
```

Comment:
I don't have a full suggestion yet, but this sentence should read:
"See <a href=\"URL>installation instructions</a>",